tests/unit_tests/shared/test_feature_toggling.py

```diff
@@ -0,0 +1,69 @@
+import os
+from unittest.mock import Mock
+
+import pytest
+
+import ert.__main__
+from ert.__main__ import ert_parser
+from ert.cli import TEST_RUN_MODE
+from ert.shared.feature_toggling import FeatureToggling
+
+
+@pytest.fixture(autouse=True)
+def mocked_valid_file(monkeypatch):
+    monkeypatch.setattr(ert.__main__, "valid_file", Mock(return_value=""))
+
+
+def test_feature_toggling_from_env(monkeypatch):
+    envs = {"ERT_ENABLE_SCHEDULER": True}
+    monkeypatch.setattr(os, "environ", envs)
```

Comment:
```suggestion
    monkeypatch.setenv("ERT_ENABLE_SCHEDULER", "1")
```

Note that `True` is not a valid value in `os.environ`.

Reply:
good catch

Reply:
I am setting all of environ to make sure that no other env vars are set. In case a developer may have set some ENABLE_/DISABLE_ in their profile or something.

Reply:
fixed